README.md

```diff
@@ -78,7 +79,7 @@ were removed from collection class and throw NotSupported error.
 
 * [aggregate(pipeline, options, callback)](http://mongodb.github.io/node-mongodb-native/2.2/api/Collection.html#aggregate)
 * [bulkWrite(operations, options, callback)](http://mongodb.github.io/node-mongodb-native/2.2/api/Collection.html#bulkWrite)
-* [count(query, options, callback)](http://mongodb.github.io/node-mongodb-native/2.2/api/Collection.html#count)
+* [count(query, options, callback)](#count)
 * [createIndex(fieldOrSpec, options, callback)](http://mongodb.github.io/node-mongodb-native/2.2/api/Collection.html#createIndex)
```

Comment:
Yep, documentation needs update, but it is barely part of current issue. I'd suggest create another issue to make documentation up to date along with code (for example check for changes in `deprecatedMethods`, replace deprecated `MongoError.create` with `MongoError` and so on)

Reply:
Well, we're now mentioning `countDocuments` and `estimatedDocumentCount` in Readme, so adding them is relevant to current issue.

And we won't be able to point those new links to 2.2 driver documentation since there are no such methods there.

And it'll look weird if some links will point to 2.2 driver version, while other will point to 3.6 version.

Reply:
But generally your're right, of course. We could create separate issue to tidy up the docs.

Reply:
Readme says that driver v2.2 is not supported, but it's still in dependencies and code still have [conditions like that](https://github.com/2do2go/mongodbext/blob/master/test/helpers.js#L31) for both versions of driver. Maybe it's a sort of deprecation, though.

Reply:
Readme point out that you can't use 2.2 driver with 3.x MongoDB. `mongodbext` supports both 2.x and 3.x driver.

Reply:
Oh, got it. Thought `since version 3.0.0` is about mongodbext

Reply:
Created issue for documentation